### DISCUSSION TITLE
MSI Cubi 5 10M (i3-10110u, i5-10210u, i7-10510u) Opencore 071 EFI

### DISCUSSION THREAD

**Discussion by reactor900** [daliansky/Hackintosh#329] [Show and tell]
https://github.com/reactor900/MSI-Cubi-5-Opencore-EFI

MSI Cubi 5 10M (10th Gen. intel core U processor)

i3-10110u, i5-10210u, i7-10510u

Opencore 071, 10.15.7 (Catalina)

Works

Ethernet
Wifi(Airportitlwm)
Bluetooth
Audio plugs
USB
HDMI (Video)
Sleep

Not works

HDMI Audio out

not tested

DP (maybe works)

**Comment by Omebs** [comment]
Hello. Thanks for sharing this. I have the Cubi 5 10m with i3 10110U. Do you know if this EFI folder can be used for Big Sur or Monterey?
